lib/util/probeUtils.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
may be these should be constants like in other components! I see that we dont have that in backbeat 7.x!

Reply:
I could move these into the constants.js if you think that is the better approach, personally not a fan of a 'constants' file as they can get quite a bit and messy over time ( reminds me of static "manager" classes from games... ).

Reply:
Yes I'd prefer constants, seems consistent. Something about calling this a util seems a bit wrong.

Reply:
yea that is fair, I sort of envisioned it having more stuff in it... but then it didn't. I will move it to the existing constants then.